src/main.cpp

```diff
@@ -1144,13 +1144,14 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
     }
 
     // Reject transactions with witness before segregated witness activates (override with -prematurewitness)
-    if (!GetBoolArg("-prematurewitness",false) && !tx.wit.IsNull() && !IsWitnessEnabled(chainActive.Tip(), Params().GetConsensus())) {
+    bool witnessenabled = IsWitnessEnabled(chainActive.Tip(), Params().GetConsensus());
```

Comment:
`s/witnessenabled/witnessEnabled` would be more consistent with all other naming, no?